.travis.yml

```diff
@@ -71,7 +71,7 @@ deploy:
   skip_cleanup: true
   file: "${RELEASE_BINDING_FILE}"
   api_key:
-    secure: GaM0z8pPpodKkBagFLQzGMzUJOl6Zj0kmhhk7FwPN4vrbCNnb9mtn9AVF/d3rTCS1yhMJ7urbR9yoC9GyYcmE6NGoIzvoT4Iha919a+/3J6cHpM0DVLfXErj1zAZSBImdhSxvnbMxSr07wAn15Dvmv1bJv+JLNE0oB51bPGEtbq9FV6An08qHOOjsPAVhi0rt/zrhtrm0MTFEWML2Kkf3rpeyqyP8RiJZkzmtJUWw/8aMmws8XOwX7TlrH/hpotHIoTSrbmcYryC+YEFbbvVXYvKBGaKFCx0xcLinG/clFqJPbS96g1JUW4ZUf96lHXxKInnkw37ujrnn7s2kjSZoRJsxqigoW8H/kwtOPzHsKHwN720bB9SHR2m6pdnfZeEx2Oc2OCqRXM430C/uH4g04t8uibZU78UVS8xgK43cFh9TZy0TYBe0Wn4/l1Tku06Xs7pyHYnigfE0ch4exL/NMsmBw/ALfkRmoR6lFuPDab3kN+4uO+VOmbYNbLlCf4wRAJNi/mMR+iw/Zv8BO9keR0UTR5cbNDo8KIjJZLo0F8rw1rY34WMgYDh52qa3q18gM41t1E625yJ5Zm6lk+wdTFrOJOjiD3zbSVfXWd2OzyCwY9D7kxTyBS+GQDh9GlTwBm/3mudkKp1xFmcoe476jFwP7XovccEbRgslv1hYtY=
+    secure: VrtFvnIa26/yCFqBBqxHV6FiSa4ERZbE1/a6wVJpsLFYKCXU1Zj9FORCL4tQKhSK7FoyiAG69NsTumKpPTBINqGSO0U8+n5YJ29Jf3BtxrrUkYjO8xHSLZLn4upcwNysCXsizUIKSZiWNw7SVB92+8xr97/Udz2ltKU7fC2mDKXPho+yGlvXSstMvO+UR+gtpUPf5WD5iWp5EAyCMFCJkpUc1233b7Vi6Aqp+7jfD6CVrOo9upXakHFu5qCmUyGbX5HTf3dV/zD+PLeeZAT86v1PS8JEpJVJxXpldy40nat8Cza9aLj5jrHF4YGyhj60ogrQd+hFVODVDaP4+NndzVrrzIy7K0plrzEV8TSRZxsI6bCYY9cCJsBJgh06Vu+SmcYrPyzWVAGwDYA+fljI4sum+hPBUQCv3WDwP+kUpQsMWzMf3lgdba5b+uGI3inX/uPFJD8LNPMeFDFrgDXNBBxlo3hh8gsy/ULGNVgvc0mw/9Sh8PTUEqMtZ3f4paCBYezqyWhumzB/swXiz6LdU211fhW9sd1g9PuuRUv7BFERkT1LQRutAHMdaO+h7LtzZYOPVLGNkoVY6WdwdW0Pd+Z7MiroFs0OV4CTY5p7PUm6E5LOKcI2VPRXlznOUYKi9qEn86Ji7I3wpWcYSuyXqasr0gIQii+VnHrmJAu3iXI=
```

Comment:
Any reason to change this key ?

Reply:
It wasn't working before.  Is that a key we set?